project2/styles.css

```
@@ -0,0 +1,134 @@
+* {
+    box-sizing: border-box;
+    /* border: 3px solid red; */
+}
+
+h2 {}
+
+.u-button-reset {
+    background: rgba(77, 28, 255, 0.527);
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+h1 {
+    font-family: 'Righteous', cursive;
+    font-size: 80px;
+    text-align: center;
+    margin: 50px 0;
+}
+
+h2 {
+    font-family: 'Overpass', sans-serif;
+    font-size: 20px;
+}
+
+nav {
+    margin-bottom: 60px;
+    font-family: 'Overpass', sans-serif;
+}
+
+ul {
+    list-style: none;
+    text-align: center;
```

By default, ordered and unordered lists have a `padding-left` value -- to get this nav truly centered, you'll want to remove that:
```suggestion
    padding-left: 0;
    text-align: center;
```